terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/FunctionTestSupport.java

```diff
@@ -44,6 +45,8 @@
 
     protected static WebDriver driver;
 
+    protected WebDriverWait webDriverWait;
```

Comment:
- [ ] It seems better to align field order & initailize order (with `webDriverOperations`).

Reply:
26d110cfb664135d5fe0cf971dbee88b750e13c2

I align order in above commit .